contents/docs/user-guides/events.mdx

```diff
@@ -36,6 +36,8 @@ PostHog detects:
 * Strings
 * Everything else
 
+If you feel something has been detected incorrectly, you can manually change the type by going to `Data Management -> Properties -> Choose event -> Edit`, and then changing the property type manually.
```

Comment:
```suggestion
If you feel something has been detected incorrectly, you can manually change the type by going to `Data management -> Properties -> Choose event -> Edit`, and then changing the property type manually.
```

@corywatilo and I haven't documented this anywhere yet that I know of, but I _believe_ we're changing casing rules so that we only Headline Case pricing tiers, not features.

Reply:
Mostly documented in our [writing style guidelines](https://posthog.com/handbook/company/communication#writing-style) but could be improved